internal/server/openapi.go

```diff
@@ -398,6 +398,20 @@ func buildRequest(r reflect.Type, op *openapi3.Operation, method string) {
 		},
 	})
 
+	op.AddParameter(&openapi3.Parameter{
+		Name:     "Authorization",
+		In:       "header",
+		Required: true,
+		Schema: &openapi3.SchemaRef{
+			Value: &openapi3.Schema{
+				Example:     "Bearer ACCESSKEY",
+				Format:      `Bearer\s[\d|a-z]{10}-[\d|a-z]{24}-`,
```

Comment:
I think it's a dot between the two sections, isn't it?
```suggestion
				Format:      `Bearer\s[\d|a-z]{10}\.[\d|a-z]{24}-`,
```

Reply:
it is. doh!

Reply:
Added the . and got rid of those dashes